moveit_ros/hybrid_planning/global_planner/global_planner_component/src/global_planner_component.cpp

```diff
@@ -63,7 +64,7 @@ bool GlobalPlannerComponent::initializeGlobalPlanner()
 {
   // Initialize global planning request action server
   global_planning_request_server_ = rclcpp_action::create_server<moveit_msgs::action::GlobalPlanner>(
-      node_, "global_planning_action",
+      node_, GLOBAL_PLANNING_ACTION_NAME,
```

Comment:
Shouldn't these be private to the node in order to allow running multiple pipelines? If yes, prepending "~" should do that

Reply:
Great. I was thinking in the ROS1 mindset here.